src/App/pages/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import Text from 'components/Text';
+import Header from 'components/Header';
+import Search from './components/Search';
+import Filter from './components/Filter';
+import Content from './components/Content';
```

Comment:
[+] можно сократить до двух строк:
```suggestion
import { Text, Header } from 'components';
import { Search, Content, Filter } from './components';
```

нужно в index.ts **внутри папки с компонентом** оставить только один импорт вида `export { default as Filter } from './Filter';` и добавить index.ts **внутри папки `components`**, откуда будут экспортироваться все компоненты этой папки в виде `export * from './Filter';`

то же в целом и к другим файликам страниц и компонентов с подобными импортами относится